.eslintrc.js

```diff
@@ -31,6 +32,7 @@ module.exports = {
   },
   plugins: ['react', '@typescript-eslint', 'prettier', 'jest'],
   rules: {
+    "prettier/prettier": "error",
```

Comment:
`'plugin:prettier/recommended'` already includes this rule. Refence: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration